src/main/java/rs117/hd/model/ModelPusher.java

```diff
@@ -386,13 +387,13 @@ private int[] getColorsForFace(Renderable renderable, Model model, ObjectPropert
             color1L = color2L = color3L = 127;
         }
 
-        if (objectProperties != null && objectProperties.isInheritTileColor()) {
+        if (objectProperties != null && objectProperties.getInheritTileColorType() != InheritTileColorType.NONE) {
             if (tile != null && (tile.getSceneTilePaint() != null || tile.getSceneTileModel() != null)) {
                 int[] tileColorHSL;
 
                 if (tile.getSceneTilePaint() != null && tile.getSceneTilePaint().getTexture() == -1) {
                     // pull any corner color as either one should be OK
-                    tileColorHSL = HDUtils.colorIntToHSL(tile.getSceneTilePaint().getSwColor());
+                    tileColorHSL = HDUtils.colorIntToHSL(tile.getSceneTilePaint().getSeColor());
```

Comment:
Was there any particular reason behind this change?

Reply:
reverting this, leftover from testing I think.